### ISSUE TITLE
add to scoop

### ISSUE THREAD

**Issue by realies** [irusanov/ZenTimings#41]

**Comment by irusanov** [comment]
@realies Hi, I saw the notification, it was something about the inpout kernel driver.
Yes, it installs itself if it's not installed already, I don't have a good mechanism to uninstall it yet. I'm working on a better handling of that driver. Meanwhile, I've prepared a batch file to completely remove it if you want to.

Run the bat file as administrator and it should stop the service, remove it, delete the file and remove the registry key. Sorry for any inconvenience and for the late response.
[Uninstall_inpout.zip](https://github.com/irusanov/ZenTimings/files/12578949/Uninstall_inpout.zip)

**Comment by realies** [comment]
@irusanov, thanks for the reply. This issue is asking for ZenTimings to be added to the scoop package manager.

**Comment by irusanov** [comment]
Yeah, sorry. There's some misunderstanding from my side. For some reason GMail notified me for some really old messages, like almost an year back.
